# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/glue/PBackground.ipdl
# Commit: cc3c39aa8fb8
# Full Hash: cc3c39aa8fb8124cca265e84f71a29cbd218b704
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-12 09:23:07
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 2 - CookieStore API - IPDL, r=edgul,webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D215145
# ==============================================================================

diff -r 2fc9a3d0ad8b -r cc3c39aa8fb8 ipc/glue/PBackground.ipdl
--- a/ipc/glue/PBackground.ipdl	Wed Sep 11 16:26:54 2024 +0000
+++ b/ipc/glue/PBackground.ipdl	Wed Sep 11 16:26:55 2024 +0000
@@ -20,6 +20,7 @@
 include protocol PCacheStorage;
 include protocol PCacheStreamControl;
 include protocol PClientManager;
+include protocol PCookieStore;
 include protocol PEndpointForReport;
 include protocol PFileSystemManager;
 include protocol PFileSystemRequest;
@@ -99,6 +100,7 @@
   manages PCacheStorage;
   manages PCacheStreamControl;
   manages PClientManager;
+  manages PCookieStore;
   manages PEndpointForReport;
   manages PFileSystemRequest;
   manages PGamepadEventChannel;
@@ -197,6 +199,8 @@
   async PUDPSocket(PrincipalInfo? pInfo, nsCString filter);
   async PBroadcastChannel(PrincipalInfo pInfo, nsCString origin, nsString channel);
 
+  async PCookieStore();
+
   async PServiceWorkerManager();
 
   async ShutdownServiceWorkerRegistrar();